homeassistant/components/risco/const.py

```diff
@@ -24,6 +24,7 @@
 CONF_HA_STATES_TO_RISCO = "ha_states_to_risco"
 CONF_COMMUNICATION_DELAY = "communication_delay"
 CONF_CONCURRENCY = "concurrency"
+CONF_CONFIG_ENTRY_ID = "config_entry_id"
```

Comment:
There's probably a preexisting const for that in HA, and it's only needed by the service, so no reason to put it here.

Reply:
I've checked that but this const doesn't exist in `homeassistant/const.py`, unless I'm looking at the wrong spot. I also don't seem to find it somewhere else in the code base except in other integrations.

Reply:
Got it, thanks.
I still think it probably belongs in the relevant file rather than here, since it's only used in one place. Where is it in other integrations that only use it in one file?

Reply:
Mostly in the consts.py. I didn't check if it is only used in one place. https://github.com/search?q=repo%3Ahome-assistant%2Fcore+%22%3D+%5C%22config_entry_id%5C%22%22&type=code
I've moved it to the `services.py` file and renamed it to `ATTR_CONF_CONFIG_ENTRY_ID` which makes more sense since it is only used to get or set an attribute.